tests/test_mock_command_handler.py

```diff
@@ -146,7 +138,7 @@ async def test_handle_command(self) -> None:
         devices_names_checked: typing.Set[str] = set()
         while len(devices_names_checked) != len(self.device_configs):
             reply = self.responses.pop()
-            name = reply[common.Key.TELEMETRY][0]
+            name = reply[common.Key.TELEMETRY][common.Key.NAME]
```

Comment:
Please update the doc string for this method (up at line 136, github doesn't allow me to add a comment there) to indicate that it not only configures the system, but also starts sending telemetry.

Reply:
Done.